src/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  margin: 0;
+  line-height: 13px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 14px 14px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.card__name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.card__code {
+  margin-bottom: 15px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-bottom: 20px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background: url("images/star-active.svg") no-repeat center;
+  border-radius: 0.5px;
+}
+
+.card__stars :last-child {
+  background: url("images/star.svg") no-repeat center;
+}
+
+.card__cost {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 20px;
+}
+
+.card__value {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__btn {
+  display: block;
+  width: 170px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 14px;
+  text-align: center;
+  line-height: 40px;
+}
+
+.card__btn:hover {
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
at this moment when you hovered on your link - border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change his position, so its better if you write border property inside the button.

Reply:
thanks, changed